catalog/scripts/gen_type_asserts.py

```
@@ -0,0 +1,81 @@
+import typing as t
+from pathlib import Path
+from textwrap import dedent
+import sys
```

Flake8 (7.2.0)

[error] 'sys' imported but unused

(F401)